initializr-service/src/main/resources/application.yml

```diff
@@ -1253,6 +1261,38 @@ initializr:
           versionRange: "[1.3.0.RELEASE,2.0.0.M1)"
           groupId: org.springframework.boot
           artifactId: spring-boot-actuator-docs
+    - name: Spring Cloud GCP
+      bom: spring-cloud-gcp
+      versionRange: 2.0.0.RC2
+      content:
+        - name: GCP Support
+          id: cloud-gcp
+          description: Spring Boot support to use GCP services through Spring Cloud
+          groupId: org.springframework.cloud
+          artifactId: spring-cloud-gcp-starter
+          links:
+            - rel: reference
+              href: https://docs.spring.io/spring-cloud-gcp/docs/1.0.0.M2/reference/htmlsingle/
+              description: Reference doc
+        - name: GCP Messaging
+          id: cloud-gcp-pubsub
+          description: GCP Pub/Sub service integration
+          groupId: org.springframework.cloud
+          artifactId: spring-cloud-gcp-starter-pubsub
+          links:
+            - rel: reference
+              href: https://docs.spring.io/spring-cloud-gcp/docs/1.0.0.M2/reference/htmlsingle/#_spring_cloud_gcp_for_pub_sub
```

Comment:
This link is dead, isn't it? One more reason to find a way to avoid hardcoding the version in here. Can't you have a `current` simlink like other projects?

Reply:
The link works for me.

I actually don't know much about documentation, since we usually just ask someone on your side to run some docs scripts to publish it.
As you can see from https://docs.spring.io/spring-cloud-gcp/docs/, there is no `current` or equivalent, unfortunately.
I seem to recall having asked for this, but can't remember what the outcome was.

Reply:
I mean the link points to `1.0.0.M2`, that's not the right version, right?

Reply:
It is the latest version indeed. When we update to M3, we'll need to manually update it here.
Not sure if I'm missing anything...

Reply:
I lost you. We need a milestone that matches what you wrote https://github.com/spring-io/initializr/pull/539#issuecomment-375747323 - Do we have that?

Reply:
So it seems I'll have to come back here and ping again when the release is out.